### ISSUE TITLE
Proposals list: testnet shows proposals as having not met the required participation where in fact they have

### ISSUE THREAD

**Issue by campbellssource** [vegaprotocol/frontend-monorepo#1320]
![Image](https://user-images.githubusercontent.com/13255539/189940224-8e88ec4d-6006-4aa6-b725-ac4344880588.png)

Click into any of these top 3 proposals and you'll see that they have met the required threshold. 

![Image](https://user-images.githubusercontent.com/13255539/189940664-09439b83-80a9-4b1b-8377-340d2b550b35.png)

at time of writing some have been enacted but did not get any more votes

**Comment by pennyandrews** [comment]
Hey @sam-keen can you pop a note in here explaining why this is blocked by core and link to the related ticket in the core repo. Ta

**Comment by sam-keen** [comment]
@pennyandrews my bad - turned out it was a wallet connect issue not a core one, https://github.com/vegaprotocol/frontend-monorepo/issues/1692. Matt's provided me with a workaround so I'll remove this as blocked for now.

**Comment by dexturr** [comment]
https://vegaprotocol.slack.com/archives/C02KVKMAE82/p1665735080152869

**Comment by dexturr** [comment]
blocked by: https://github.com/vegaprotocol/vega/issues/6504

**Comment by dexturr** [comment]
blocked by: https://github.com/vegaprotocol/vega/issues/6356

**Comment by pennyandrews** [comment]
@gordsport the core ticket Dex created https://github.com/vegaprotocol/vega/issues/6504 could well be a duplicate of the existing ticket https://github.com/vegaprotocol/vega/issues/6356 but it would be great if you were able to check and remove whichever one you think makes sense if you agree they are duplicates. Ta

**Comment by dexturr** [comment]
The core issue here has been resolved, there are no required FE actions except those covered in #1766 so closing this ticket
